python-threatexchange/threatexchange/cli/dataset_cmd.py

```diff
@@ -43,7 +43,14 @@ def init_argparse(cls, ap) -> None:
             "-t",
             dest="only_type",
             metavar="STR",
-            help="only process one type of indicators (or signals)",
+            help="only process one type of indicator (eg HASH_MD5), signal (eg photo_md5), or content (eg photo); specifying a signal or content type implies '--signal-summary'",
+        )
+        ap.add_argument(
+            "--tag",
+            "-g",
+            dest="only_tag",
+            metavar="STR",
+            help="only process a single tag; implies '--signal-summary'",
```

Comment:
thanks for updating the docstrings! this strikes mea as more of a 'filter' than a 'process' but just a minor nit especially given this matches the wording of `--type`